src/cargo/ops/cargo_compile.rs

```diff
@@ -47,7 +47,7 @@ pub struct CompileOptions<'a> {
     pub all_features: bool,
     /// Flag if the default feature should be built for the root package
     pub no_default_features: bool,
-    /// Root package to build (if empty it's the current one)
+    /// A set of packages to build.
```

Comment:
@steveklabnik should I end docs for fields/enum variants with a period or not? This particular struct is painfully inconsistent in this matter :)